src/entities/Mob.ts

```diff
@@ -21,6 +27,8 @@ export class Mob {
         setInterval(() => {
             this.updateMoveRandomly()
         }, 1000);
+
+        this.isFiring = true;
```

Comment:
With removeMesh, you don't need this property anymore. I think. Because garbage collector should remove the object since the program has no dependency with it so it won't fire anyway. Not sure. Have to test

Reply:
frerot c'est outdated